libraries/Microsoft.Bot.Schema/CardImage.cs

```diff
@@ -30,9 +30,7 @@ public CardImage(string url = default, string alt = default, CardAction tap = de
         /// <summary>Gets or sets URL thumbnail image for major content property.</summary>
         /// <value>The URL of the thumbnail.</value>
         [JsonProperty(PropertyName = "url")]
-#pragma warning disable CA1056 // Uri properties should not be strings
-        public string Url { get; set; }
-#pragma warning restore CA1056 // Uri properties should not be strings
+        public Uri Url { get; set; }
```

Comment:
Are there any Serialization tests for this class?
I'm concerned that the Bot Schema, which is used to render Activities must not be changed. At least, the serialized result should not change. 

https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#cardimage-object
![image](https://user-images.githubusercontent.com/805847/152207706-2159db54-3ad9-4091-8478-f81d8d58f232.png)

Reply:
Done. Thanks!